### ISSUE TITLE
SA1010 and SA1026 are reported for the same whitespace error

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#250]
In the following code, SA1026 is reported for the `new` keyword and SA1010 is reported for the `[` token. In this situation, SA1010 should be suppressed.

``` csharp
var array = new [] { 1 };
```
